graph_generation/attribute_generator.py

```diff
@@ -77,12 +77,14 @@ def __init__(self, dataset_params, system_params, data_integrity_params, process
     @staticmethod
     def generate_time(seconds_range, n=1):
```

Comment:
Is this used to generate random SLOs? I think this should return things like "13d", "34h", "344m", "43s", but never a combination of multiple things, eg "1d 3h 5m". Especially if it's for SLOs, where the granularity is high enough

Reply:
This is used for all the time generation (including data integrity - restoration, regeneration and recreation times, and SLO). Is SLO and time defined by the highest granularity (ex. 1d 3h 5m would just be 1d)?

Reply:
You can think of it like significant numbers. All of these (SLO, restoration time, etc) would never be 1d 3h 5m, because it doesn't make sense to have a 5m precision on a 1d+ duration. It would instead be 27h. Other example: 5h 30 would be 330m.

Reply:
Okay, makes sense, will change.